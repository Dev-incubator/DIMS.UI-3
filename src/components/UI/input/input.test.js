import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Input from './Input';

configure({
  adapter: new Adapter(),
});

describe('<Input/>', () => {
  it('render Input element', () => {
    const wrapper = shallow(<Input />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
