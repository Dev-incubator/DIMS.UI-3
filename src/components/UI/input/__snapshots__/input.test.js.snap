// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Snapshot Input element 1`] = `
<div
  className="form-group "
>
  <label />
  <input
    autoComplete="on"
    disabled={false}
    placeholder=""
    type="text"
  />
</div>
`;
